README.md

```diff
@@ -1,6 +1,102 @@
 # Agriboard
+
+Agriboard is an administrative dashboard for the agriculture software stack. It is used to manage users and view advanced analytics.
+
+## Dependencies
+
+### NodeJS
+
+To get started, the first thing you're going to need NodeJS (version `8.9.0` or higher). You can download this [here](https://nodejs.org/en/)
+
+### Yarn
+
+You will also require Yarn to manage the javascript dependencies in the application. You can view installation instructions for your OS [here](https://yarnpkg.com/lang/en/docs/install/)
+
+To install dependencies simply run `yarn`
+
+**Important:** Please use yarn to manage your dependencies. **You should never run any of the following commands:**

```

Comment:
nice! maybe we should have this in the other readmes as well?

Reply:
Yeah I think it's fairly important